Tests daniel

danielgpz opened the pull request:
Los tests de semantica, son 52 en total. Creo que bastantes simples y recorren todo el contenido del manual de dicha parte.
Claro esta, puede tener errores que segun salgan a la luz ire arreglando. Creo que la gran mayoria estan bien elaborados o eso espero :).

danielgpz committed:
Fix bugs

Use TimeoutExpired instead of TimeoutError

danielgpz committed:
Add first semantic test

danielgpz committed:
Update utils

danielgpz committed:
Add new tests

danielgpz committed:
Add new tests

danielgpz committed:
Add new tests

danielgpz committed:
Add dispatch tests

danielgpz committed:
Add conditionals tests

danielgpz committed:
Add loops test

danielgpz committed:
Add blocks tests

danielgpz committed:
Add let tests

danielgpz committed:
Add case tests

danielgpz committed:
Add isvoid tests

danielgpz committed:
Add aritmetics tests

danielgpz committed:
Add equility tests

danielgpz committed:
Format all semantic errors

apiad commented:
Hay algo raro con los tests de Travis. Vamos a intentar hacerlo directo con Github Action?

leynier commented:
@apiad @C0NKER yo cree eso en mi fork para trabajar más cómodo

https://github.com/codestrange/cool-compiler-2020/blob/tests/.github/workflows/tests.yml

leynier commented:
Por si les sirve de algo

apiad commented:
> Por si les sirve de algo

Seguro!!!! Prueba a ver @leynier si estos te tests corren también, y si es así, please, dame un pull request con el .yaml :pray:

danielgpz committed:
Update travis.yml

danielgpz committed:
Add basic types tests

danielgpz commented:
> Hay algo raro con los tests de Travis. Vamos a intentar hacerlo directo con Github Action?

Ya vi lo que pasa, olvide poner en el `travis.yml` el comando para correr los nuevos tests. Ya hice un commit con eso y 8 tests mas que habia olvidado.

danielgpz committed:
New arithmetic tests

danielgpz committed:
Add new case tests

danielgpz committed:
Add new type tests

danielgpz committed:
Add codegen test engine

danielgpz committed:
Update clean rule of makefile

danielgpz committed:
Add codegen tests

apiad commented:
@leynier necesitamos ayuda para echar a andar los tests de generación en Github Actions, y de hecho quitarnos a Travis del medio. Si puedes colaborar con esto, sería de gran ayuda.

apiad committed:
Merge branch 'entrega-parser' into testsDaniel

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 853.43kb -> 707.58kb (17.09%)

/img/img1.png -- 92.73kb -> 67.11kb (27.64%)
/img/img3.png -- 19.09kb -> 15.33kb (19.71%)
/img/img12.png -- 31.49kb -> 25.40kb (19.34%)
/img/img11.png -- 37.65kb -> 31.08kb (17.44%)
/img/img8.png -- 62.78kb -> 52.08kb (17.04%)
/img/img6.png -- 44.09kb -> 36.61kb (16.98%)
/img/img9.png -- 59.13kb -> 49.41kb (16.44%)
/img/img10.png -- 309.29kb -> 259.23kb (16.18%)
/img/img7.png -- 71.18kb -> 60.18kb (15.45%)
/img/img4.png -- 32.15kb -> 27.47kb (14.54%)
/img/img5.png -- 10.59kb -> 9.11kb (14.01%)
/img/img2.png -- 83.25kb -> 74.57kb (10.42%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

leynier committed:
Add GitHub Action workflow

leynier committed:
Add spim installation

leynier committed:
Fix typo

leynier committed:
Merge pull request #1 from leynier/imgbot

[ImgBot] Optimize images

leynier committed:
Merge pull request #1 from leynier/imgbot

[ImgBot] Optimize images

leynier committed:
Merge branch 'entrega-parser' into feature/github-action-tests

apiad committed:
Merge pull request #43 from leynier/feature/github-action-tests

[Feature] GitHub Action Tests
